src/scripts/main.js

```diff
@@ -21,7 +21,20 @@ const food = {
 const tree = document.querySelector('#tree');
 
 function createTree(element, data) {
-  // WRITE YOUR CODE HERE
+  const ul = document.createElement('ul');
```

Comment:
```suggestion
   if (!data || typeof data !== 'object') {
    return null;
  }
  
  const ul = document.createElement('ul');
```
add such check before func logic. it can help to pass tests.